python/tests/conftest.py

```diff
@@ -53,9 +72,11 @@ def simple_ts():
     return msprime.simulate(2, random_seed=42)
 
 
-# A tree sequence with data in all fields
 @fixture(scope="session")
 def ts():
```

Comment:
This is great, but I worry a bit about attaching meaning to the (very commonly used) variable ``ts`` here. I can see this being confusing for people. What if we adopt a convention like having a suffix or prefix of ``fixture`` for these parameters? That might make things a bit more obvious.

Reply:
Yeah, this kind of magic can be really confusing if you aren't aware what it going on. It seemed a bit much to put a suffix on, but when I think about the reader of the code it really does help. I've added them in https://github.com/tskit-dev/tskit/pull/909/commits/b0b566b26220b9a9214b378c3163b5586b341814

Reply:
I agree, it feels a bit wordy in the code, but I think it'll help people who aren't used to using pytest.